svc-fabrics/README.md

```diff
@@ -627,7 +627,7 @@ curl -i GET \
 ```
 { 
    "@odata.id":"/redfish/v1/Fabrics/8f6d8828-a21a-464f-abf9-ed062fa08cd9/Endpoints/b21f3e57-e46d-4a8e-92c8-8658edd107cb",
-   "@odata.type":"#Endpoint.v1_11_0.Endpoint",
+   "@odata.type":"#Endpoint.v1_7_0.Endpoint",
```

Comment:
why are we going back in version?

Reply:
Previously was updated wrong, That is correct